Makefile

```diff
@@ -88,7 +90,8 @@ $(NOVENDOR_PACKAGES):
 	$(GO_GET) $@
 
 $(VENDOR_PATH):
-	$(GLIDE) install
+	$(GLIDE) install; \
+	rm -rf vendor/github.com/Sirupsen/;
```

Comment:
Why is this needed?

Reply:
https://github.com/sirupsen/logrus/issues/570#issuecomment-313933276
Will be deleted soon.